src/components/Header.jsx

```diff
@@ -1,11 +1,26 @@
-import Topics from './topics/Topics';
-import NavBar from './NavBar';
+import { Link } from 'react-router-dom';
+import Topics from './Topics';
 
 const Header = () => {
 	return (
 		<>
 			<h1>NC News</h1>
-			<NavBar />
+			<nav className="navbar navbar-expand-lg bg-body-tertiary">
+				<div className="container-fluid">
+					<Link className="navbar-brand" to="/">
+						Home
+					</Link>
+
+					<ul className="navbar-nav ml-auto">
+						{' '}
+						<li className="nav-item">
+							<Link className="btn btn-outline-success" to="/login">
```

Comment:
This route does not exist so it shouldn't be here. It takes the user to a page they can't get to